requirements.txt

```diff
@@ -0,0 +1,13 @@
+atomicwrites==1.4.0
```

Comment:
This looks to require too many dependencies that is not used in the library. If they are not imported directly it is good to omit from this.

Reply:
Done.